PaywallViewControllerDelegateWrapper.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PaywallViewControllerWrapper.swift
+//  PurchasesHybridCommonUI
+//
+//  Created by Cesar de la Vega on 2/2/24.
+//
+
+import Foundation
+import RevenueCat
+import RevenueCatUI
+
+/// Delegate for ``PaywallViewController`` that sends dictionaries instead of the original objects.
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
+@objc(RCPaywallViewControllerDelegateWrapper)
+public protocol PaywallViewControllerDelegateWrapper: AnyObject {
```

Comment:
Used in https://github.com/RevenueCat/react-native-purchases/pull/887/files#diff-4c098c9f4efa9edf7579aae6e47a211ce08715082ee3b1009876e90423346117R76

Reply:
I am not sure this is the right approach or if there's anything better we can do, so open for feedback

Reply:
Yeah not much we can do to prevent the duplication.